README.md

```diff
@@ -263,11 +263,12 @@ describe 'myclass' do
 end
 ```
 
-2. In your `spec_helper.rb`
+You can also globally set facts.
```

Comment:
I think it's worthwhile to mention that this would generally go in `spec_helper.rb`